crates/libeq_wld/src/lib.rs

```diff
@@ -463,7 +463,7 @@ mod tests {
 
     #[test]
     fn it_builds_meshes() {
-        let wld_data = &include_bytes!("../fixtures/gfaydark.wld")[..];
+        let wld_data = &include_bytes!("../../../fixtures/gfaydark.wld")[..];
```

Comment:
Oh no! Hrmmm, what OS are you running? I created a symlink to `fixtures` at `crates/libeq_wld/fixtures` as a way to avoid all of the `../../../`. But I'm not totally sure how/if that works on all OSe 😬.

Reply:
ah I see. in this case I was on windows.  not sure there's a universal solution here.  I think to support symlinks like this you have to make a script that converts these into windows style ones

Reply:
hm there may be some way to make this work I'll give it a look.  windows is such a mess :)

Reply:
Aha: https://stackoverflow.com/questions/5917249/git-symbolic-links-in-windows/59761201#59761201

Working now!

Reply:
Cool! Glad it works. If it causes too many problems though we can always ditch the symlink and use relative paths :D.